config/locales/es/general.yml

```
@@ -933,3 +934,21 @@ es:
   machine_learning:
     comments_summary: "Resumen de comentarios"
     info_text: "Contenido generado mediante IA / Machine Learning"
+  cookies_consent:
+    accept_all_cookies: "Aceptar todo"
+    accept_essential_cookies: "Aceptar cookies esenciales"
+    message: "Las cookies nos ayudan a ofrecer nuestros servicios. Al utilizar nuestros servicios, aceptas el uso de cookies."
+    more_info_link: "Más información sobre cookies"
+    management: "Ajustes"
+    title: "Política de cookies"
+  cookies_management:
+    title: Configuración de cookies
+    description: Esta web utiliza cookies propias y de terceros para su correcto funcionamiento. Al hacer click en el botón Aceptar todo, acepta el uso de estas tecnologías y el procesamiento de tus datos para estos propósitos.
+    essentials:
+      title: Cookies esenciales
+      description: Utilizadas para finalidades muy diversas, como por ejemplo, reconocerte como usuario, elegir el idioma o personalizar la forma en la que se muestra el contenido. Sin estas las funciones básicas de navegación se verán perjudicadas, por eso deben estar siempre activas.
```

In English, we aren't displaying the last part. So, for consistency:

```suggestion
      description: Utilizadas para finalidades muy diversas, como por ejemplo, reconocerte como usuario, elegir el idioma o personalizar la forma en la que se muestra el contenido.
```